github.xverizex.nem-desktop.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "github.xverizex.nem_desktop",
```

Comment:
```suggestion
    "app-id" : "io.github.xverizex.nem_desktop",
```

Reply:
that also mean renaming the manifest file.

Reply:
renaming with io.?

Reply:
Do I need to do this myself?

Reply:
got it. I made.